code/modules/uplink/uplink_item.dm

```diff
@@ -181,7 +181,7 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	desc = "The support specialist: Aid your fellow operatives with this medical bundle. Contains a Donksoft machine gun, \
 			a box of ammo, and a pair of magboots to rescue your friends in no-gravity environments."
 	item = /obj/item/weapon/storage/backpack/dufflebag/syndie/med/medicalbundle
-	cost = 15 // normally 20
+	cost = 10 // normally 14
```

Comment:
This isn't a part of normal hippiecode, carbon. This is the sales code. /tg/ code has a sales setup, where they offer this for cheaper. Hence the comment //normally 14. This is the sales code